app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+    <background android:drawable="@color/ic_launcher_background"/>
```

Comment:
You might be able to change this to `@color/colorPrimary` and remove the `ic_launcher_background.xml` file as a result. Can you give that a try and let me know if it works please? 😄

Reply:
Yup! Just switched it over now